pm_icecon/bt/compute_bt_ic.py

```diff
@@ -872,7 +872,7 @@ def calc_bootstrap_conc(
     return ic_perc
 
 
-def bootstrap_for_cdr(
```

Comment:
This function more generically wraps the bootstrap alg. This can continue to evolve, but it's not specific to the cdr anymore.